Server - INFO - initialization complete
CON-1 LOGGER - INFO - Connection successfully initiated, beginning handshake
Server - INFO - Handshake starting
CON-1 LOGGER - INFO - Sending data of type StartupData to the client
CON-1 LOGGER - INFO - Sending data of type AuthRequest to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type AuthRequest
CON-1 LOGGER - INFO - Sending data of type AuthRequest to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type AuthRequest
https://icicle.tapis.io
Server - INFO - initiated in 4.290422201156616
CON-1 LOGGER - INFO - Sending data of type AuthRequest to the client
Server - INFO - Final connection data sent
CON-1 LOGGER - INFO - Handshake complete
CON-1 LOGGER - INFO - Sending command arguments now
CON-1 LOGGER - INFO - Sending data of type ResponseData to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type ResponseData
CON-1 LOGGER - INFO - Argument setup succeeded on the client side
CON-1 LOGGER - INFO - Asyncio task for socket operations was received, and is now active
CON-1 LOGGER - INFO - All connection setup complete, beginning normal operations
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type CommandData
{'connection': <server.server.ServerConnection object at 0x0000023E3C79B2B0>,
 'expression': True,
 'help': False,
 'id': 'cicatalog',
 'positionals': [],
 'verbose': False}
CON-1 LOGGER - INFO - Attempting to execute the command neo4j
'NAME: expression\nVALUE: True\n REQUIRED: True'
CON-1 LOGGER - INFO - Sending data of type FormRequest to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type FormResponse
CON-1 LOGGER - INFO - Sending data of type ResponseData to the client
CON-1 LOGGER - WARNING - Traceback (most recent call last):
  File "C:\Users\ahuma\Desktop\programming\python_programs\hello_icicle_auth_clients\icicle_rel_03_2023\CLI\TapisCL-ICICLE\src\TapisCLICICLE\server\server.py", line 194, in receive_and_execute
    result = await self.run_command(connection, kwargs)
  File "C:\Users\ahuma\Desktop\programming\python_programs\hello_icicle_auth_clients\icicle_rel_03_2023\CLI\TapisCL-ICICLE\src\TapisCLICICLE\commands\commandMap.py", line 258, in run_command
    return await command(**kwargs)
  File "C:\Users\ahuma\Desktop\programming\python_programs\hello_icicle_auth_clients\icicle_rel_03_2023\CLI\TapisCL-ICICLE\src\TapisCLICICLE\commands\baseCommand.py", line 340, in __call__
    raise e
  File "C:\Users\ahuma\Desktop\programming\python_programs\hello_icicle_auth_clients\icicle_rel_03_2023\CLI\TapisCL-ICICLE\src\TapisCLICICLE\commands\baseCommand.py", line 318, in __call__
    return_value = await self.run(**kwargs)
  File "C:\Users\ahuma\Desktop\programming\python_programs\hello_icicle_auth_clients\icicle_rel_03_2023\CLI\TapisCL-ICICLE\src\TapisCLICICLE\commands\query\neo4j.py", line 21, in run
    return_value = graph.run(kwargs['expression'])
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\database.py", line 405, in run
    return self.auto().run(cypher, parameters, **kwparameters)
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\database.py", line 992, in run
    self._connector.pull(result, -1)
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\client\__init__.py", line 1434, in pull
    cx.pull(result, n=n)
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\client\bolt.py", line 1001, in pull
    self._audit(self._transaction)
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\client\bolt.py", line 810, in _audit
    task.audit()
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\client\bolt.py", line 1140, in audit
    item.audit()
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\client\bolt.py", line 1140, in audit
    item.audit()
  File "C:\Users\ahuma\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py2neo\client\bolt.py", line 1303, in audit
    raise self._failure
py2neo.errors.ClientError: [Statement.SyntaxError] Invalid input '{': expected <init> (line 1, column 1 (offset: 0))
"{'expression': ''}"
 ^

CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type CommandData
{'canRunBatch': False,
 'connection': <server.server.ServerConnection object at 0x0000023E3C79B2B0>,
 'defaultAuthnMethod': None,
 'description': None,
 'effectiveUserId': None,
 'enableCmdPrefix': False,
 'help': True,
 'importRefId': None,
 'jobCapabilities': None,
 'jobConstraints': None,
 'jobEnvVariables': None,
 'jobRuntimes': None,
 'jobWorkingDir': None,
 'mountDataTransferNode': None,
 'mpiCmd': None,
 'notes': None,
 'port': None,
 'positionals': [],
 'systemId': None,
 'tags': None,
 'useProxy': False,
 'verbose': False}
CON-1 LOGGER - INFO - Attempting to execute the command update_system
CON-1 LOGGER - INFO - Sending data of type ResponseData to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type CommandData
{'canRunBatch': False,
 'connection': <server.server.ServerConnection object at 0x0000023E3C79B2B0>,
 'defaultAuthnMethod': None,
 'description': None,
 'effectiveUserId': None,
 'enableCmdPrefix': False,
 'help': False,
 'importRefId': None,
 'jobCapabilities': None,
 'jobConstraints': None,
 'jobEnvVariables': None,
 'jobRuntimes': None,
 'jobWorkingDir': None,
 'mountDataTransferNode': None,
 'mpiCmd': True,
 'notes': None,
 'port': None,
 'positionals': ['TESTING'],
 'systemId': None,
 'tags': None,
 'useProxy': False,
 'verbose': False}
CON-1 LOGGER - INFO - Attempting to execute the command update_system
'NAME: mpiCmd\nVALUE: True\n REQUIRED: False'
CON-1 LOGGER - INFO - Sending data of type FormRequest to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type FormResponse
CON-1 LOGGER - INFO - Sending data of type ResponseData to the client
CON-1 LOGGER - INFO - Waiting to receive data from the client on this connection
CON-1 LOGGER - INFO - Successfully received data of type CommandData
{'connection': <server.server.ServerConnection object at 0x0000023E3C79B2B0>,
 'help': False,
 'positionals': [],
 'verbose': False}
CON-1 LOGGER - INFO - Attempting to execute the command shutdown
CON-1 LOGGER - WARNING - shutdown initiated
['OPEN']
CON-1 LOGGER - INFO - ATTEMPTING TO CLOSE
CON-1 LOGGER - INFO - successfully cancelled task
CON-1 LOGGER - INFO - Sending data of type ResponseData to the client
[]
CON-1 LOGGER - INFO - SUCCESSFULLY CLOSED
Server - INFO - [CancelledError(''), None, Shutdown('shutdown initiated')]
